.github/actions/binding_setup/action.yml

```diff
@@ -48,6 +48,13 @@ runs:
           ccache --set-config "hash_dir=false"
           ccache --zero-stats
 
+    # This appears to be needed to work around this issue:
+    # https://github.com/gagolews/stringi/issues/486
+    - name: "Manually install stringi"
+      if: inputs.lang == 'R'
+      shell: bash
+      run: Rscript -e "options(configure.args='--disable-pkg-config'); renv::install('stringi')"
```

Comment:
Alternative you could force the `libicu70` share library package in.  Demo:

```sh
Rscript -e "install.packages('https://p3m.dev/cran/__linux__/jammy/latest/src/contrib/stringi_1.8.4.tar.gz?r_version=4.4&arch=x86_64', repos = NULL)"
Rscript -e 'library(stringi)    # fails
wget http://mirrors.kernel.org/ubuntu/pool/main/i/icu/libicu70_70.1-2_amd64.deb
dpkg -i libicu70_70.1-2_amd64.deb
Rscript -e 'library(stringi)'    # now works
```

which I just did in a container.  This is a bug in the p3m.dev binaries you use.  (I have built something that works more tightly via proper packages, see https://eddelbuettel.github.io/r2u but this PR is not the time to switch.

Reply:
Oh, it would be great to switch to r2u.  For now though it feels just a little hacky to install an old version of libicu70 alongside the existing system version... so let me try one or two other things to get stringi to build correctly, and failing that I'll use the workaround you posted.

Reply:
Ok I give up, I'll use your workaround :smile: